dist/tools/packer/riot.json

```diff
@@ -85,11 +85,11 @@
     "cpus": "1",
     "disk_size": "65536",
     "headless": "true",
-    "iso_checksum": "f529548fa7468f2d8413b8427d8e383b830df5f6",
-    "iso_checksum_type": "sha1",
-    "iso_name": "ubuntu-16.04.2-server-amd64.iso",
+    "iso_checksum": "a06cd926f5855d4f21fb4bc9978a35312f815fbda0d0ef7fdc846861f4fc4600",
+    "iso_checksum_type": "sha256",
+    "iso_name": "ubuntu-16.04.3-server-amd64.iso",
     "iso_path": "/Volumes/Storage/software/ubuntu",
-    "iso_url": "http://releases.ubuntu.com/16.04/ubuntu-16.04.2-server-amd64.iso",
+    "iso_url": "http://releases.ubuntu.com/16.04/ubuntu-16.04.3-server-amd64.iso",
```

Comment:
This has nothing to do with importing Keccak. Please provide a seperate PR for this vagrant box version bump.